openssh/USAGE.md

```diff
@@ -0,0 +1,128 @@
+
+
+## Quick start 
+
+How to set up your quantum-safe OpenSSH is described in the corresponding [README.md](README.md).
+
+## Limitations
+
+For limitations of the post-quantum algorithms themselves, refer to [this](https://github.com/open-quantum-safe/openssh#limitations-and-security).
+
+## Slightly more advanced usage options
+
+### Change install location
+
+The OQS-OpenSSH binaries, configuration files and host keys are located in the default install location `/opt/oqs-ssh/`. This location can be changed at build time with `--build-arg INSTALL_DIR=</path/to/new/location>`.
+
+### Access the man pages of oqs-ssh
+
+Man pages for oqs-ssh are installed in `<INSTALL_DIR>/share/man` and can be viewed by for example `man -l <INSTALL_DIR>/share/man/man1/ssh.1` or `man -l <INSTALL_DIR>/share/man/man5/ssh.5`. Note that those man pages are **not** different from the original ssh man pages. So it might be easier to consult them on your local system or the internet. 
+
+Direct links to man pages: [ssh(1)](https://linux.die.net/man/1/ssh), [sshd(8)](https://linux.die.net/man/8/sshd), [ssh_config(5)](https://linux.die.net/man/5/ssh_config), [sshd_config(5)](https://linux.die.net/man/5/sshd_config)
+
+## Seriously more advanced usage options
+
+### Choosing the algorithms
+
+For a list of all algorithms see [here](https://github.com/open-quantum-safe/openssh#supported-algorithms). It is recommended to only use hybrid algorithms to maintain established classical security. The post-quantum safe algorithms have not yet received enough confidence to be relied on as the only security mechanism.
+
+The image's default key-exchange algorithm is `ecdh-nistp384-kyber-1024-sha384@openquantumsafe.org` for key-exchange with `curve25519-sha256` for backwards compatibility. For host and identity key (authentication) algorithms `ssh-p256-dilithium2` and classical `ssh-ed25519` are used. Those algorithms may be changed by adjusting the files `ssh_config` and `sshd_config` respectively. In the built image, those files are located at the default install location (see above). After changing something in `sshd_config`, the sshd must be restarted using `rc-service oqs-sshd restart`. Alternatively, the configuration can be changed **pre**-build by changing [ssh_config](ssh_config) or [sshd_config](sshd_config) and rebuilding the image.
```

Comment:
I don't understand  "backward compat" means in this sentence: "The image's default key-exchange algorithm is `ecdh-nistp384-kyber-1024-sha384@openquantumsafe.org` for key-exchange with `curve25519-sha256` for backwards compatibility." The former alg is already an hybrid; what fallback does x25519 provides?

Reply:
The fallback to standard ssh. But thinking about it, as this *is* supposed to be a quantum-safe demo of ssh, it does not make much sense to support that. You can always fall back on the standard ssh installation if available.

Reply:
I see (and I understood later after reading more of the documentation). Fallbacks might be problematic in security (e.g., downgrade attacks), and since this is indeed a PQ project, one would assume there is a need for the PQ algs. One scenario could be a server serving multiple clients, some PQ, some not; but perhaps we should explain how to add a classical-only scheme vs. making it a default.

Reply:
Yes indeed. I will adapt this.